app/views/exam_templates/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for exam_template,
```

Comment:
Use `form_for [@assignment, exam_template]` here - this will allow you to remove the `url` argument.